src/coredump/coredump.c

```diff
@@ -756,6 +763,9 @@ static int submit_coredump(
         if (core_message)
                 IOVEC_SET_STRING(iovec[n_iovec++], core_message);
 
+        if (truncated)
+                IOVEC_SET_STRING(iovec[n_iovec++], "COREDUMP_TRUNCATED=yes");
```

Comment:
Hmm, my guess is that journal fields should be primarily computer readable, not human-readable, hence I think assigning "1" instead of "yes" might be more appropriate?